packages/slo-stat-bot/test/slo-stat-label.ts

```diff
@@ -24,7 +24,7 @@ import {describe, it, beforeEach, afterEach} from 'mocha';
 import Webhooks from '@octokit/webhooks';
 import snapshot from 'snap-shot-it';
 import handler from '../src/slo-stat-label';
-import sinon from 'sinon';
+const sinon = require ('sinon');
```

Comment:
This shouldn't pass the linter check.  This tells me maybe we never added this repo to `ci.yaml`, where we (sadly) today need to manually add new bots/directories.